io_context/include/msg_converters/example_interfaces.hpp

```diff
@@ -31,12 +31,16 @@ namespace common
 
 inline void from_msg(const UInt8MultiArray & in, std::vector<uint8_t> & out)
 {
-  std::copy(in.data.begin(), in.data.end(), out.begin());
+  for (const auto & byte : in.data){
+    out.push_back(byte);
+  }
```

Comment:
While this works, it has the disadvantage of allocating memory for every byte one-at-a-time. The preferred method is to use `.reserve()` and then `std::copy` (the `.reserve()` is missing, which is what causes problems here). Please refactor.